examples/next-ui/app/entry.server.tsx

```diff
@@ -0,0 +1,23 @@
+import { renderToString } from "react-dom/server";
+import { RemixServer } from "remix";
+import type { EntryContext } from "remix";
+import { NextUIProvider, getCssText } from "@nextui-org/react";
+export default function handleRequest(
+  request: Request,
+  responseStatusCode: number,
+  responseHeaders: Headers,
+  remixContext: EntryContext
+) {
+  const markup = renderToString(
+    <NextUIProvider>
+    <RemixServer context={remixContext} url={request.url} />
+    </NextUIProvider>
+  ).replace(/<\/head>/, `<style id="stitches">${getCssText()}</style></head>`);
```

Comment:
Why is the `id` here "stitches"? Does nextui depend on stitches?

Reply:
Yes it does